core/Cargo.toml

```diff
@@ -36,6 +36,8 @@ all-features = true
 [features]
 default = [
   "rustls",
+  "services-redis-rustls",
```

Comment:
Please don't enable this service by default. Other LGTM.

Reply:
I am sorry, I think i misunderstood what you meant with the redis feature and the answer to my previous question about selecting `services-redis-rustls` by default.

How would you like it to work ?

I can see 3 options:

1. Just having 3 features, and the user would choose only one one them. Everyone already using `services-redis` would not automatically get TLS support
```
services-redis = ["dep:redis"]
services-redis-rustls = ["redis/tokio-rustls-comp"]
services-redis-native-tls = ["redis/tokio-native-tls-comp"]
```

2. Having `service-redis` to activate a no-tls redis and the other one that would just add the tls part (therefore the user would need to choose at least 2 features to get redis + tls). Everyone already using `services-redis` would not automatically get TLS support
```
services-redis = ["dep:redis"]
services-redis-rustls = ["redis?/tokio-rustls-comp"]
services-redis-native-tls = ["redis?/tokio-native-tls-comp"]
```

3. Integrating the tls in the `service-redis` (so everyone already using this feature would silently get TLS support), and having an alternative one to get the other TLS implementation (and maybe a notls option ?)
```
services-redis = ["redis/tokio-rustls-comp"]
services-redis-native-tls = ["redis/tokio-native-tls-comp"]
# services-redis-notls = ["dep:redis"]
```